src/global_logic.js

```diff
@@ -2860,8 +2860,12 @@ module.exports.treatSupportAbility = function (totals, chara, buff) {
                     totals[key]["ougiDamageLimitBuff"] += support.value;
                     continue;
                 case "additionalDamageXA":
-                    // currently, range: own only, and no chances to stack.
-                    totals[key]["additionalDamageXA"] = support.value;
+                    // Implemented Stacking. Can someone verify?
```

Comment:
Stacking:
Support ability of Levin Sisters and Jeanne d'Arc (Grand) does not coexist. (Levin Sisters is adding 15% when TA.)
https://gbf.wiki/Levin_Sisters

 info:
https://xn--bck3aza1a2if6kra4ee0hf.gamewith.jp/article/show/176457

Reply:
reverted till verified